### PR TITLE
Recently Closed Tabs

### PR DESCRIPTION
**Pull request by sarinali**
# Description

Ensures that the recently closed tabs function works in the panel menu

## Bugzilla bug ID

**ID:** 1895900
**Link:** https://bugzilla.mozilla.org/show_bug.cgi?id=1895900

## Type of change

- [x] New Test

# How does this resolve / make progress on that bug?

Finishes

# Screenshots / Explanations

N/A

# Comments / Concerns

Maybe a random link generator would make this test more arbitrary in the future

### CODE REVIEW

File: modules/util.py [hunk 1]
```diff
@@ -394,15 +396,20 @@ def find_shadow_chrome_element(
         self, nodes: list[WebElement], selector: list
     ) -> Union[WebElement, None]:
         logging.info("Selecting element in Chrome Context Shadow DOM...")
-        if selector[0] != By.ID:
+        if selector[0] not in self.allowed_selectors:
+            # if selector[0] != By.ID:
             raise ValueError(
-                "Currently shadow elements in chrome can only be selected by ID."
+                "Currently shadow elements in chrome can only be selected by ID and classname."
```
Comment (ben-c-at-moz) [inline modules/util.py:402]: Didn't we create functionality for selecting shadow chrome elements by tag name? What happened to that code?

File: tests/tabs/test_recently_closed_tabs.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import logging
+
+from selenium.webdriver import Firefox
+from selenium.webdriver.support import expected_conditions as EC
+
+from modules.browser_object import PanelUi, TabBar
+
+links = [
```
Comment (ben-c-at-moz) [inline tests/tabs/test_recently_closed_tabs.py:8]: I might consider only using sites where you know the load latency is very small. If you can confirm that is the case for all of these, go for it. If not, you can open a bunch of about pages instead.

**Review by ben-c-at-moz — CHANGES_REQUESTED** [review]
These are small things, but I would actually love to see them addressed. Thanks!

File: modules/util.py (hunk 1 above)
Comment (sarinali) [reply modules/util.py:402]: i think i discarded the changes bc it wasn't relevant atm to the shadow element. in the end, it was for the pin tab PR and that was figured out with a workaround

Comment (ben-c-at-moz) [reply modules/util.py:402]: OK, that's fine

**Pull request closed without merging**